assets/globals.d.ts

```diff
@@ -62,52 +64,53 @@ interface IClientConfig {
     multimedia_website_search_url?: string;
 }
 
-interface Window {
-    sectionNames: {
-        home: string;
-        wire: string;
-        agenda: string;
-        monitoring: string;
-        saved: string;
-    };
+declare global {
+    interface Window {
+        sectionNames: {
+            home: string;
+            wire: string;
+            agenda: string;
+            monitoring: string;
+            saved: string;
+        };
 
-    newsroom: {
-        client_config: IClientConfig;
-        websocket?: any;
-    };
+        newsroom: {
+            client_config: IClientConfig;
+            websocket?: any;
+        };
 
-    locale: any;
-    gtag: any;
-    analytics: any;
-    google: any;
-    iframely: any;
-    translations: any;
-    profileData: any;
-    wireData: any;
-    homeData: any;
-    agendaData: any;
-    amNewsData: any;
-    viewData: any;
-    report: any;
-    factCheckData: any;
-    companyReportsData: any;
-    mapsLoaded: any;
-    googleMapsKey: any;
-    mediaReleasesData: any;
-    notificationData: any;
-    twttr: any;
-    instgrm: any;
-    locales: any;
-    marketPlaceData: {
-        home_page: any;
-    };
-    restrictedActionModalBody: string;
+        locale: any;
+        gtag: any;
+        analytics: any;
+        google: any;
+        iframely: any;
+        translations: any;
+        profileData: any;
+        wireData: any;
+        homeData: any;
+        agendaData: any;
+        amNewsData: any;
+        viewData: any;
+        report: any;
+        factCheckData: any;
+        companyReportsData: ICompanyReportsData;
```

Comment:
I think you can just declare this part of the data in the `company-reports/index` and keep this file unmodified, it should merge it with this declaration:
```ts
declare global {
    interface Window {
        companyReportsData: ...
    }
}
```

Reply:
and keep it with `any`?

Reply:
I will give it a try.

Reply:
yeah, you're right. I misunderstood.